eng/pipelines/mirror-repos.yml

```diff
@@ -63,3 +63,14 @@ jobs:
             TargetRepos: 
               Azure/azure-cli-extensions-pr:
               azure-sdk/azure-cli-extensions:
+
+    - template: ./templates/steps/sync-repo-branch.yml
```

Comment:
You shouldn't need to add the template again. You should only need to add the repo configuration.

Reply:
But you cannot add duplicated key to a yaml object

Reply:
I thought we could but if that is actually an error I'm fine with this solution for now. If the duplicate key is trouble though we should consider another way to configure multiple branches. I'll try to look further into that later.